widgets/extctrls.pas

```diff
@@ -190,7 +190,11 @@ implementation
 { TCustomWebSocketClient }
 
 function TCustomWebSocketClient.WebSocketMessageHandler(aEvent: TEventListenerEvent): Boolean;
+var
+  s: TJSMessageEvent;
 begin
+  s := TJSMessageEvent(aEvent);
+  writeln(s.data);  // <- how get type? for text -
```

Comment:
I have problem with this place. 
I cannot get the type of message.
For text message in debuger i see:
![image](https://user-images.githubusercontent.com/6418554/117865785-ff231c00-b29e-11eb-8e73-a7e35fe43d6f.png)

For binary message in debuger i see
![image](https://user-images.githubusercontent.com/6418554/117865888-1d891780-b29f-11eb-95f6-de019aa1fd37.png)